<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDraw">
       <attribute name="title">
        <string>Draw</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="minimumSize">
           <size>
            <width>770</width>
            <height>570</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>770</width>
            <height>570</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutToolsDraw">
          <item>
           <widget class="QTabWidget" name="tabWidgetDraw">
            <property name="tabShape">
             <enum>QTabWidget::Triangular</enum>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <widget class="QWidget" name="tabDrawLine">
             <attribute name="title">
              <string>Draw Line</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayoutDrawLine">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayoutStartPoint">
                  <item>
                   <widget class="QLabel" name="labelA">
                    <property name="text">
                     <string>Start Point</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QFormLayout" name="formLayoutStartPoint">
                    <item row="0" column="0">
                     <widget class="QLabel" name="labelAx">
                      <property name="text">
                       <string>x</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="labelAy">
                      <property name="text">
                       <string>y</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QSpinBox" name="spinBoxStartX">
                      <property name="maximum">
                       <number>770</number>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QSpinBox" name="spinBoxStartY">
                      <property name="maximum">
                       <number>570</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="Line" name="linePoints">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayoutEndPoint">
                  <item>
                   <widget class="QLabel" name="labelB">
                    <property name="text">
                     <string>End Point</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QFormLayout" name="formLayoutEndPoint">
                    <item row="0" column="0">
                     <widget class="QLabel" name="labelBx">
                      <property name="text">
                       <string>x</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="labelBy">
                      <property name="text">
                       <string>y</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QSpinBox" name="spinBoxEndX">
                      <property name="maximum">
                       <number>770</number>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QSpinBox" name="spinBoxEndY">
                      <property name="maximum">
                       <number>570</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonDrawLine">
                <property name="text">
                 <string>Draw Line</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabDrawSun">
             <attribute name="title">
              <string>Draw Sun</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayoutSunPoint">
                <item>
                 <widget class="QLabel" name="labelSunCenter">
                  <property name="text">
                   <string>Sun Center Point</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayoutSun">
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelSunCenterX">
                    <property name="text">
                     <string>x</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelSunCenterY">
                    <property name="text">
                     <string>y</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="spinBoxSunCenterX">
                    <property name="maximum">
                     <number>770</number>
                    </property>
                    <property name="value">
                     <number>385</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="spinBoxSunCenterY">
                    <property name="maximum">
                     <number>570</number>
                    </property>
                    <property name="value">
                     <number>285</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayoutSunParametr">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelSunAngleStep">
                  <property name="text">
                   <string>Δθ</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelSunRadius">
                  <property name="text">
                   <string>R</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinBoxSunRadius">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                  <property name="value">
                   <number>200</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBoxAngleStep">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>360</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonDrawSun">
                <property name="text">
                 <string>Draw Sun</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutColor">
            <item>
             <layout class="QVBoxLayout" name="verticalLayoutColorLabel">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayoutColorLine">
                <item>
                 <widget class="QLabel" name="labelTextColorLine">
                  <property name="text">
                   <string>Line</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelColorLine">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonChangeColorLine">
                  <property name="text">
                   <string>Change</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayoutColorBackGround">
                <item>
                 <widget class="QLabel" name="labelTextColorBackGround">
                  <property name="text">
                   <string>Background</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelColorBackGround">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonChangeColorBackGround">
                  <property name="text">
                   <string>Change</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutColorButton">
              <item>
               <widget class="QPushButton" name="pushButtonSwapColor">
                <property name="text">
                 <string>Swap</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonClear">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxPainters">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTime">
       <attribute name="title">
        <string>Time</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutTabTime"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRungs">
       <attribute name="title">
        <string>Rungs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutTabRungs"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
